azure-pipelines-master.yml

```diff
@@ -33,12 +33,134 @@ steps:
     workingFolder: tools/Deployment
 
 - task: EsrpCodeSigning@1
-  displayName: Sign executables and dlls
+  displayName: Sign docfx executables and dlls
   inputs:
     ConnectedServiceName: 'CodeSigning-APEX'
     FolderPath: '$(Build.SourcesDirectory)/target/Release'
-    signType: 'batchSigning'
-    batchSignPolicyFile: '$(Build.SourcesDirectory)/codesign_policy.json'
+    Pattern: |
+        **\Microsoft.DocAsCode.*.dll
+        **\*.exe
+    UseMinimatch: true
+    signConfigType: inlineSignParams
+    inlineOperation: |
+     [
+        {
+          "KeyCode": "CP-230012",
+          "OperationSetCode": "SigntoolSign",
+          "parameters": [
+            {
+              "parameterName": "OpusName",
+              "parameterValue": "Microsoft"
+            },
+            {
+              "parameterName": "OpusInfo",
+              "parameterValue": "http://www.microsoft.com"
+            },
+            {
+              "parameterName": "PageHash",
+              "parameterValue": "/NPH"
+            },
+            {
+              "parameterName": "TimeStamp",
+              "parameterValue": "/tr \"http://rfc3161.gtm.corp.microsoft.com/TSS/HttpTspServer\" /td sha256"
+            },
+            {
+              "parameterName": "FileDigest",
+              "parameterValue": "/fd \"SHA256\""
+            }
+          ],
+          "ToolName": "sign",
+          "ToolVersion": "1.0"
+        },
+        {
+          "KeyCode": "CP-230012",
+          "OperationSetCode": "SigntoolVerify",
+          "Parameters": [
+            {
+              "parameterName": "VerifyAll",
+              "parameterValue": "/all"
+            }
+          ],
+          "ToolName": "sign",
+          "ToolVersion": "1.0"
+        }
+      ]
+    SessionTimeout: '60'
+    MaxConcurrency: '50'
+    MaxRetryAttempts: '5'
+
+- task: EsrpCodeSigning@1
+  displayName: Sign dependent dlls
+  inputs:
+    ConnectedServiceName: 'CodeSigning-APEX'
+    FolderPath: '$(Build.SourcesDirectory)/target/Release'
+    Pattern: |
+        **\DotLiquid.resources.dll
```

Comment:
Can we use the exclude syntax here? My concern is that we may forget to update this when add a dependency in the future. We should try avoid missing signing files.

Reply:
let me see

Reply:
Updated. Please review.